permission_validators/parameter/new/src/lib.rs

```diff
@@ -0,0 +1,95 @@
+//! Validator that checks [`NewParameter`] instruction
+//! and respective [`Grant`] and [`Revoke`] instructions.
+
+#![no_std]
+#![no_main]
+
+extern crate alloc;
+
+use iroha_wasm::{
+    data_model::prelude::*,
+    validator::{pass_conditions, prelude::*},
+};
+
+/// Strongly-typed representation of `can_grant_permission_to_create_parameters` permission token.
+#[derive(Token, Validate, Clone, Copy)]
+#[validate(pass_conditions::AlwaysDeny)] // Can be granted only in genesis
```

Comment:
```suggestion
#[validate(pass_conditions::OnlyGenesis)]
```

Reply:
Will fix in my another openned PR: #3226